src/App.tsx

```diff
@@ -1,26 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const total = getNumbers(1, Math.ceil(items.length / perPage));
+
+  const firstElement = (currentPage - 1) * perPage + 1;
+  const lastElement = Math.min(currentPage * perPage, items.length);
+  const visibleElements = items.slice(firstElement - 1, lastElement);
+
+  // eslint-disable-next-line max-len
+  const handleSelectItemsPerPage = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
```suggestion
  const handleSelectItemsPerPage = (
    event: React.ChangeEvent<HTMLSelectElement>
  ) => {
```
Try this to avoid disabling the eslint